### PR TITLE
Switch from embedded index to Elasticsearch server 

### PR DESCRIPTION
**Pull request by fsteeg**
Will resolve https://github.com/hbz/lobid-gnd/issues/67

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -23,11 +23,10 @@ entityfacts {
 }
 
 index {
-	home: "."
-	name: "authorities"
+	hosts: ["localhost"] # ["weywot3.hbz-nrw.de", "weywot4.hbz-nrw.de", "weywot5.hbz-nrw.de"]
```
Comment (dr0i) [inline conf/application.conf]: localhost seems to be potentially harmful - depending on what server you start the program and if this is really taken into acount?

Comment (dr0i) [reply conf/application.conf]: ah, ok . I also overlooked the '#' . So +1

**Comment by fsteeg** [comment]
> I vote to artificially make indexing slower. What do you think @fsteeg?

Added a 500ms break after each 1000-item bulk request, which takes about 500-1500ms each.

Used to be about 200ms with embedded index on quaoars. Looking forward to the SSDs.

**Comment by dr0i** [comment]
+1

**Pull request merged**